int32 Max_Trunk
int32 Min_Trunk
float64 Trunk_corr
int32 Max_Ankle
int32 Min_Ankle
float64 Ankle_corr
float64 err_tilde